halo2_gadgets/src/fibonacci.rs

```diff
@@ -208,29 +208,29 @@ mod tests {
 
     #[test]
     fn test_full_prove() {
-        use halo2_curves::secp256k1::{Fq, Secp256k1Affine};
         use halo2_proofs::plonk::*;
         use halo2_proofs::poly::commitment::Params;
         use halo2_proofs::transcript::{Blake2bRead, Blake2bWrite, Challenge255, EncodedChallenge};
         use rand::rngs::OsRng;
+        use secpq_curves::secp256k1::{Affine, Scalar};
```

Comment:
@vb7401  Isn’t it `use secpq_curves::secp256k1::{Affine, Base};`?

And use `Base` (i.e. Fp of secp, Fq of secq) for witness values?

Reply:
I think the witness math all happens in the Scalar field, which is Fq for secp.

Reply:
@vb7401 Don't we need to fit in Fp of secp to witness math so we can represent points like public key, G, etc?

Reply:
Just seeing this -- I think we discussed in our call but just so we have it on paper, I only have a working IPA impl for secp because its 2-adicity is high enough to run a small circuit (k < 6). Need to implement ECFFT for secq if we want to use it, but likely that MSM is the right path forward.